packages/app-desktop/gui/ResizableLayout/ResizableLayout.tsx

```diff
@@ -112,14 +111,18 @@ function ResizableLayout(props: Props) {
 		function onResizeStart() {
 			setResizedItem({
 				key: item.key,
-				initialWidth: sizes[item.key].width,
-				initialHeight: sizes[item.key].height,
+				initialSize: sizes[item.key].current,
+				maxSize: sizes[item.key].max,
+				minSize: sizes[item.key].min,
 			});
 		}
 
 		function onResize(_event: any, direction: string, _refToElement: any, delta: any) {
-			const newWidth = Math.max(itemMinWidth, resizedItem.initialWidth + delta.width);
-			const newHeight = Math.max(itemMinHeight, resizedItem.initialHeight + delta.height);
+			let newWidth = Math.max(resizedItem.minSize.width, resizedItem.initialSize.width + delta.width);
+			let newHeight = Math.max(resizedItem.minSize.height, resizedItem.initialSize.height + delta.height);
+
+			newWidth = Math.min(newWidth, resizedItem.maxSize.width);
+			newHeight = Math.min(newHeight, resizedItem.maxSize.height);
```

Comment:
It turns out these lines weren't even necessary. I just figured they might be needed because the minimum size was also applied here, but everything seems to work fine without adding these.